kyuubi-rest-client/pom.xml

```diff
@@ -33,13 +33,79 @@
 
     <properties>
         <jars.target.dir>${project.build.directory}</jars.target.dir>
+        <httpclient.version>4.5.13</httpclient.version>
+        <junit.version>4.13.2</junit.version>
```

Comment:
seems that the junit.version is also used in kyuubi-hive-jdbc module, we can unify them.